core/admin_menu/class-btm-admin-page-notification-single.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+if ( ! defined( 'BTM_PLUGIN_ACTIVE' ) ) {
+	exit;
+}
+
+/**
+ * Class BTM_Admin_Task_Single_Page
+ */
+final class BTM_Admin_Notification_Single_Page {
+
+	/**
+	 * BTM_Admin_Task_Single_Page constructor.
+	 */
+	public function __construct() {
+		$this->add_submenu_page();
+	}
+
+	// region Page
+
+	protected function add_submenu_page() {
+		$page_hook = add_submenu_page(
+			'admin.php',
+			__( 'Notification Rule', 'background_task_manager' ),
```

Comment:
@eriknovembit let's just call it `Notifications`

Reply:
I fix it